### ISSUE TITLE
[Meta] How to communicate off-topic ideas

### ISSUE THREAD

**Issue by certik** [j3-fortran/fortran_proposals#109]
@FortranFan asked in https://github.com/j3-fortran/fortran_proposals/issues/108#issuecomment-564770183:

> May I please see some consistent adherence and application of this?

See also @milancurcic's reply in https://github.com/j3-fortran/fortran_proposals/issues/108#issuecomment-564786945.

The consistent suggestion that I have is to simply look at all members who participate here and react to your posts, and if you see that some are getting discouraged to participate here, then please step back and try to find a way to communicate your ideas without people getting annoyed. I provided some suggestions how to do that, the main one being to simply create a new issue for the off topic idea and just link it. That way the main technical thread does not get derailed. When I say "you", I really mean everybody including myself.

@FortranFan, we can discuss this in detail here in this issue. And people who are not interested can ignore it.

**Comment by FortranFan** [comment]
@certik, can this forum use 'we/us' instead of "you" the next time something refers to "everybody including" the author?

**Comment by certik** [comment]
@FortranFan yes, I modified the post. I couldn't figure out how to best formulate it. Thanks for the suggestion, I agree.

**Comment by FortranFan** [comment]
https://github.com/j3-fortran/fortran_proposals/issues/92#issuecomment-569315654 is off-topic as far as I'm concerned (I explain briefly in https://github.com/j3-fortran/fortran_proposals/issues/92#issuecomment-569315654).  I only see it try to "derail the discussion" in that thread or get some "discouraged to participate here".

**Comment by FortranFan** [comment]
In https://github.com/j3-fortran/fortran_proposals/issues/92#issuecomment-569303364, @gronki wrote:

> This last proposal looks as if Fortran 77 nightmare was trying to creep back in. In my opinion function attributes are most intuitive before the function name as it is currently.

The first sentence in this comment can only try to "derail the discussion" and get "some discouraged from participating here".

As to the overall comment, it ignores RESULT and BIND came after FORTRAN 77.  Besides, a phrase such as "Fortran 77 nightmare was trying to creep back in" by @gronki is needless hyperbole (https://github.com/j3-fortran/fortran_proposals/issues/108#issuecomment-564728548) and is entirely objectionable.

Also, that "last proposal" mentioned therein was only trying to suggest an alternative which expands on the approach already adopted by standard committee with RESULT, BIND to see if some solution can be arrived at to help OP of that thread with the suggested idea.
